docs/source/schema/complex_types/record.rst

```diff
@@ -2,11 +2,10 @@
  record
 ########
 
-The ``record`` type represents a labeled array capable of holding any data type, just like the
-struct in C++ or the Series in pandas.
+The ``record`` type is where we can definte complext data stuctures by grouping related variables togather in the same place. It is simiilar to 
+``struct`` in C++ or the ``Series`` in pandas. It is preferred to use `record` to hold the grouped data in each row, and a column, in portex, is a series of records of the same type.
```

Comment:
please make the commit message follow the [angular commit message format]( message:https://github.com/angular/angular/blob/main/CONTRIBUTING.md#-commit-message-format)

Also make it more detailed

Reply:
```
`record` -> ``record``
 use ``xxx`` syntax in rst file for code samples
```

Reply:
Done. Could you give me an example on what is the preferred header for this commit? Thx

Reply:
For me, the message of this commit can be:
```
docs: polish the words in "record.rst"
```